bigframes/operations/semantics.py

```diff
@@ -27,6 +29,169 @@ def __init__(self, df) -> None:
 
         self._df = df
 
+    def agg(
+        self,
+        instruction: str,
+        model,
+        cluster_column: typing.Optional[str] = None,
+        max_agg_rows: int = 10,
```

Comment:
Would it be better to call this "batch_size" or something similar?

Reply:
I used `batch_num` initially but think `max_agg_rows` would be more concise to describe its details.